app/models/form526_submission.rb

```diff
@@ -19,7 +19,7 @@ def start(klass)
     workflow_batch = Sidekiq::Batch.new
     workflow_batch.on(
       :success,
-      'EVSS::DisabilityCompensationForm::SubmitForm526#workflow_complete_handler',
+      'Form526Submission#workflow_complete_handler',
       'submission_id' => id
     )
     jids = workflow_batch.jobs do
```

Comment:
Just for future reference it may be important to comment here that there will only ever be one job in this batch.

Reply:
Which is why we only pass back `jids.first`